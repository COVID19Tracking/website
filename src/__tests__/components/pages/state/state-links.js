import React from 'react'
import renderer from 'react-test-renderer'
import StateLinks from '../../../../components/pages/state/state-links'

describe('Components : Pages : State : State links', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <StateLinks
          name="California"
          twitter="california-covid"
          dataSource="https://example.com"
        />,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()

    const covid19SiteTree = renderer
      .create(
        <StateLinks
          name="California"
          twitter="california-covid"
          covid19Site="https://example.com"
          dataSource="https://example.com"
        />,
      )
      .toJSON()
    expect(covid19SiteTree).toMatchSnapshot()
  })
})
