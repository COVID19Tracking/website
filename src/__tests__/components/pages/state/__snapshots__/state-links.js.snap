// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Pages : State : State links renders correctly 1`] = `
<ul
  className="list-unstyled listUnstyled"
>
  <li>
    <a
      href="https://twitter.com/california-covid"
    >
      California
       on Twitter
    </a>
  </li>
  <li>
    <a
      href="https://example.com"
    >
      Data source
    </a>
  </li>
</ul>
`;

exports[`Components : Pages : State : State links renders correctly 2`] = `
<ul
  className="list-unstyled listUnstyled"
>
  <li>
    <a
      href="https://twitter.com/california-covid"
    >
      California
       on Twitter
    </a>
  </li>
  <li>
    <a
      href="https://example.com"
    >
      Best current data source
    </a>
  </li>
  <li>
    <a
      href="https://example.com"
    >
      Data source
    </a>
  </li>
</ul>
`;
